### PR TITLE
Use linked-hash-map on Github repo

### PR DESCRIPTION
**Pull request by zonyitoo**
For dev version of `lru-cache`, it could use the dev version of `linked-hash-map`.

### CODE REVIEW

**Comment by FlashCat** [comment]
r? @reem

(I've picked a reviewer for you, use r? to override)

File: Cargo.toml [hunk 1]
```diff
@@ -15,4 +15,4 @@ keywords = ["data-structures"]
 readme = "README.md"
 
 [dependencies]
-linked-hash-map = "*"
+linked-hash-map = { git = "https://github.com/contain-rs/linked-hash-map" }
```
Comment (apasel422) [inline Cargo.toml:18]: I'm not in favor of changing this to a dependency on the repo. I'll publish a new version of `linked-hash-map` shortly.

Comment (zonyitoo) [inline Cargo.toml:18]: And you also need to publish a new version of `lru-cache`, because of the deprecation of `HashState`.

Comment (apasel422) [inline Cargo.toml:18]: Do you want to amend this line to use version `0.0.8` of `LinkedHashMap`?

**Comment by zonyitoo** [comment]
I am going to close this and reopen a new PR from my master branch.

**Pull request closed without merging**